<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebSniff</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="gbPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Port</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HTTP Port</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteHttpPortList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteHttpsPortList">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HTTPS Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCapture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Capture</string>
    </property>
    <widget class="QRadioButton" name="rbWinDivert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>WinDivert</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbArpSpoof">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ARPSpoof</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbWinDivertOption">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Option</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbArpSpoofOption">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Option</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbProxy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Proxy</string>
    </property>
    <widget class="QLabel" name="HTTP">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leHttpInPort">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leHttpOutPort">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leHttpsOutPort">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="HTTPS">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SSL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leHttpsInPort">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Proxy Process Name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteProxyProcessNameList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbOutboundDataChange">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Outbound</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbInboundDataChange">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Inbound</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leStripOutPort">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="SSLStrip">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Strip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leStripInPort">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbWrite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>381</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Write</string>
    </property>
    <widget class="QCheckBox" name="chkDump">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dump</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkWriteAdapter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Adapter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDumpFilePath">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxAdapterIndex">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbSslStrip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>SSL Strip</string>
    </property>
    <widget class="QCheckBox" name="chkSslStripEnabled">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Virtual IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Domain Prefix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSslStripVirtualIp">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSslStripDomainPrefix">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbRun">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>600</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pteHttpPortList</tabstop>
  <tabstop>pteHttpsPortList</tabstop>
  <tabstop>rbWinDivert</tabstop>
  <tabstop>pbWinDivertOption</tabstop>
  <tabstop>rbArpSpoof</tabstop>
  <tabstop>pbArpSpoofOption</tabstop>
  <tabstop>pteProxyProcessNameList</tabstop>
  <tabstop>leHttpInPort</tabstop>
  <tabstop>leHttpOutPort</tabstop>
  <tabstop>leHttpsInPort</tabstop>
  <tabstop>leHttpsOutPort</tabstop>
  <tabstop>leStripInPort</tabstop>
  <tabstop>leStripOutPort</tabstop>
  <tabstop>pbOutboundDataChange</tabstop>
  <tabstop>pbInboundDataChange</tabstop>
  <tabstop>chkSslStripEnabled</tabstop>
  <tabstop>leSslStripVirtualIp</tabstop>
  <tabstop>leSslStripDomainPrefix</tabstop>
  <tabstop>chkDump</tabstop>
  <tabstop>leDumpFilePath</tabstop>
  <tabstop>chkWriteAdapter</tabstop>
  <tabstop>cbxAdapterIndex</tabstop>
  <tabstop>pbRun</tabstop>
  <tabstop>pbSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
